src/describescreen.cpp

```diff
@@ -428,6 +428,26 @@ void TextWindow::DescribeSelection() {
             double d = (p1.Minus(p0)).Dot(n0);
             Printf(true,  "      distance = %Fi%s", SS.MmToString(d).c_str());
         }
+    } else if(gs.n == 3 && gs.faces == 3) {
+        Printf(false, "%FtTHREE PLANE FACES");
+
+        Vector n0 = SK.GetEntity(gs.face[0])->FaceGetNormalNum();
```

Comment:
What is wrong with a for loop?

Reply:
I "fixed" this in https://github.com/solvespace/solvespace/pull/1258/commits/c59e93c3e37a0b3e07d9346f4087fa2a7264798d